packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -378,6 +402,74 @@ class RenderEditable extends RenderBox {
     return newOffset;
   }
 
+  // Handles shortcut functionality including cut, copy, paste and select all
+  // using control + (X, C, V, A).
+  void _handleShortcuts(int pressedKeyCode) async {
+    switch(pressedKeyCode) {
```

Comment:
nit: missing space after `switch`. (You want a space after keywords, to distinguish them from method calls.)